.gitignore

```diff
@@ -33,6 +33,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 *.pyc
 *~
+src/**/*.mod
```

Comment:
How come we have .mod files under `src`?

Reply:
If we really do then I think this could just be `*.mod`.

Reply:
It's from vscode, which creates .mod files automatically as part of the code parsing. Since they shouldn't be part of ./src/, it's better to directly ignore them.

Reply:
Ah, I see. That's fine then.